linode/instanceconfig/schema_resource.go

```diff
@@ -233,24 +230,88 @@ var helpersSchema = map[string]*schema.Schema{
 	},
 }
 
+const (
+	onlyAllowedForVPCMsg  = "This attribute is only allowed for VPC interfaces."
+	onlyAllowedForVLANMsg = "This attribute is only allowed for VLAN interfaces."
+	requiredForVPCMsg     = "This attribute is required for VPC interfaces."
+	requiredForVLANMsg    = "This attribute is required for VPC interfaces."
```

Comment:
Probably need to update the description for `requiredForVLANMsg`?

Reply:
Oops, thanks for catching it!